flask_socketio/__init__.py

```diff
@@ -70,6 +70,15 @@ def process_event(self, packet):
                 app = self.request
                 return self.socketio._dispatch_message(app, self, message, args)
 
+            def get_room(self, room):
+                if not self.socketio.rooms.get(self.ns_name, False):
+                    return []
+                return list(self.socketio.rooms[self.ns_name].get(room, []))
```

Comment:
I'm sorry, I thought:

> But I would be okay with exposing the list of rooms and the list of clients in a room through functions that I can later reimplement (...)

Meant to place some stubs with the old structures that could later be reimplemented.

But that's alright, I can workaround using the proposed `socket.sessid`. I'll be back with another patch.